packages/style-engine/src/styles/color/background.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import type { Style, StyleOptions } from '../../types';
+import { generateRule, getSlugFromPreset } from '../utils';
+
+const text = {
+	name: 'background',
+	generate: ( style: Style, options: StyleOptions ) => {
+		return generateRule(
+			style,
+			[ 'color', 'background' ],
+			'backgroundColor',
+			options
+		);
+	},
+	getClassNames: ( style: Style ) => {
+		const classNames = [];
+		const styleValue: string | undefined = get( style, [
```

Comment:
Can we get away with optional chaining here instead of using `get`?

Reply:
Good call. Probably everywhere too. It was a mindless copy paste from the existing code.